phi/vectordb/cassandra/cassandra.py

```diff
@@ -0,0 +1,144 @@
+from typing import Optional, List, Dict, Any, Iterable
+from phi.vectordb.base import VectorDb
+from phi.embedder import Embedder
+from phi.document import Document
+from .index import PhiMetadataVectorCassandraTable
```

Comment:
Fix the import here. It should look something like this `from phi.vectordb.cassandra.index import PhiMetadataVectorCassandraTable`